src/components/SpecialResolution/SigningParty.vue

```diff
@@ -123,6 +146,9 @@ export default class SigningParty extends Vue {
     familyName: '',
     additionalName: ''
   }
+  // Used for undo for corrections.
+  signingDateOriginal = ''
+  signatoryOriginal = {}
```

Comment:
Can you give this a type, please?

Also, does it have to be an empty object, or could it be null?

Reply:
Thanks, but now signatoryOriginal is left undefined, so it won't be reactive. Better to assign null or `{} as PersonIF`.

Reply:
It isn't needed to be reactive - it's programmatically copied from a method

Reply:
Thanks.